backend/app.ts

```diff
@@ -3,8 +3,17 @@
 // require("regenerator-runtime/runtime");
 import "regenerator-runtime/runtime";
 
-// Run backend with interval cache updates.
+// Run backend with cache updates.
 const { updateLumensCache } = require("./routes");
+const { updateLedgers } = require("./ledgers");
 
-setInterval(updateLumensCache, 10 * 60 * 1000);
-updateLumensCache();
+(async () => {
+  setInterval(updateLumensCache, 10 * 60 * 1000);
+  console.log("starting lumens cache update");
```

Comment:
Do we want to keep these `console.log` here and below?

Reply:
i'd like to ya , when looking at the kube logs it helps with debugging/monitoring to see when the updates are started (since not at the same time now due to Horizon rate limiting)